src/features.ts

```diff
@@ -34,6 +34,15 @@ export const OPT_IN_FEATURES: Feature[] = [
     get: env().isForcedProductionView,
     set: env().forceProductionView,
   },
+  {
+    name: 'Timezones',
+    preferenceName: 'enableTimezones',
+    active: true,
+    enabled: false,
+    allowInternalProduction: false,
+    description: ['Allow choosing timezone'],
```

Comment:
time zone (that's our preference as per the PRD)

Reply:
Good catch! Thanks